HelpSource/Classes/ToolBar.schelp

```diff
@@ -0,0 +1,118 @@
+TITLE:: ToolBar
+summary:: An inline button bar populated by MenuActions
+categories:: GUI
+related:: Classes/Menu, Classes/MenuAction, Classes/ToolBar, Classes/MainMenu
+
+DESCRIPTION::
+A ToolBar is similar to a link::Classes/Menu::, with several key differences:
+LIST::
+	## ToolBars act as regular link::Classes/View::s, and can be placed in layouts or in other views.
+	## MenuActions on a toolbar appear as buttons instead of being listed menu-style
+	## By default, MenuActions that have icons show ONLY the icon. MenuActions with text will appear as a text button. This can be changed using the toolButtonStyle property.
+::
+
+code::
+(
+View().fixedWidth_(300).layout_(HLayout(
+	ToolBar(
+		MenuAction("Option A", { "Option A".postln }),
+		MenuAction("Option B", { "Option B".postln }),
+		MenuAction("Option C", { "Option C".postln })
+	)
+)).front
+)
+::
+
+CLASSMETHODS::
+
+METHOD:: new
+	Create a new ToolBar containing one or more actions.
+
+	ARGUMENT::
+		One or more link::Classes/MenuAction::
+
+INSTANCEMETHODS::
+
+METHOD:: orientation
+	The horizontal or vertical orientation of the ToolBar
+	ARGUMENT::
+		A link::Classes/QOrientation::
+
+METHOD:: toolButtonStyle
+	Controls whether the toolbar shows icons, text, or both.
+	ARGUMENT::
+		A link::Classes/QToolButtonStyle::
+	DISCUSSION::
+		CODE::
+(
+~icon = Image(64).draw({ Pen.fillOval(Rect(0, 0, 64, 64)) });
+~styles = QToolButtonStyle.classVarNames;
+
+~tool = ToolBar(
+	MenuAction("Text")
+		.icon_(~icon)
+		.action_({
+			~tool.toolButtonStyle = (~tool.toolButtonStyle + 1) % ~styles.size;
+			~styles[~tool.toolButtonStyle].postln;
+		})
+).front;
+)
+		::
+
+EXAMPLES::
+
+code::
+(
```

Comment:
this example doesn't work here. 

```
ERROR: Message 'enabled_' not understood.
RECEIVER:
Instance of MenuAction {    (0x129b4d098, gc=80, fmt=00, flg=00, set=02)
  instance variables [4]
    qObject : RawPointer 0x7f90ed71d9c0
    finalizer : instance of Finalizer (0x1178aec08, size=2, set=1)
    virtualSlots : nil
    action : instance of Function (0x129b4cf78, size=2, set=2)
}
ARGS:
   false
CALL STACK:
	DoesNotUnderstandError:reportError
		arg this = <instance of DoesNotUnderstandError>
	Nil:handleError
		arg this = nil
		arg error = <instance of DoesNotUnderstandError>
	Thread:handleError
		arg this = <instance of Thread>
		arg error = <instance of DoesNotUnderstandError>
	Object:throw
		arg this = <instance of DoesNotUnderstandError>
	Object:doesNotUnderstand
		arg this = <instance of MenuAction>
		arg selector = 'enabled_'
		arg args = [*1]
	< closed FunctionDef >  (no arguments or variables)
	AbstractMenuAction:onTriggered
		arg this = <instance of MenuAction>
		arg b = false
^^ The preceding error dump is for ERROR: Message 'enabled_' not understood.
RECEIVER: MenuAction("start")
```

Reply:
it works for me, can you check again?

Reply:
never mind, I have an issue with it now as well

Reply:
fixed now